cpp/include/raft/linalg/gemv.h

```diff
@@ -25,6 +25,38 @@
 namespace raft {
 namespace linalg {
 
+template <typename math_t, bool DevicePointerMode = false>
+void gemv(const raft::handle_t& handle,
+          const bool trans_a,
```

Comment:
And here, to also point out the ability to accept a pointer on device for alpha and beta